.php_cs-fixer.dist.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
Please rename to `.php-cs-fixer.dist.php` ;)

It's wrong in the app generator. I've sent a pull request there, but it's complicated. ;)